integrations/tensorflow/e2e/slim_vision_models/slim_vision_model_test.py

```diff
@@ -0,0 +1,88 @@
+# Lint as: python3
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test all vision models from slim lib."""
+
+import posixpath
+
+from absl import app
+from absl import flags
+import numpy as np
+from pyiree.tf.support import tf_test_utils
+from pyiree.tf.support import tf_utils
+import tensorflow as tf
+import tensorflow_hub as hub
+
+FLAGS = flags.FLAGS
+
+# Testing vision models from
+# https://github.com/tensorflow/models/tree/master/research/slim
+# slim models were designed with tf v1 and then coverted to SavedModel
+# they are stored at tensorflow_hub. At least a subset can be viewed here:
+# https://tfhub.dev/s?dataset=imagenet&module-type=image-classification,image-classifier
+flags.DEFINE_string(
+    'model_tag', 'mobilenet_v1_100_224', 'examples of model tag: '
+    'resnet_v1_50, resnet_v1_101, resnet_v2_50, resnet_v2_101, '
+    'mobilenet_v1_100_224, mobilenet_v1_025_224, mobilenet_v2_100_224, '
+    'mobilenet_v2_035_224')
+
+HUB_URL = 'https://tfhub.dev/google/imagenet/'
```

Comment:
Please make external dependencies like this controlled by command line flags

Reply:
Done.